src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/account-roles-list.js

```diff
@@ -28,22 +28,16 @@ angular.module('platformWebApp')
             return _.any(blade.currentEntities, function (x) { return x.$selected; });
         }
 
+        $scope.selectButtonClick = (row) => {
+            row.$selected = !row.$selected
+        };
+
```

Comment:
Could this be done in template, not in the controller?

Reply:
done